src/contact/components/DisplayInfo.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+
+class DisplayInfo extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+  
+  componentDidMount() {
+    if(this.props.info === null){
+      this.props.actions.displayInfo(false);
+    }
+  }
+
+  onEdit(){
+    this.props.history.push('/edit');
+  }
+
+  render() {
+    //fallback UI
+    if (this.props.hasError || this.props.info === null) {
+      return (
+        <div>
```

Comment:
Why do you need a div here? Even if you need can we have a class specifying the need of the block? Let classnames provide symantic meaning to your code.

Reply:
ok